setup.py

```diff
@@ -50,7 +50,13 @@
     extras_require=dict(
         tsne=["fitsne"],
         leiden=["leidenalg"],
-        mkl=["mkl"]
+        louvain=["louvain"],
+        scanorama=["scanorama"],
+        gprofiler=["gprofiler-official"],
+        torch=["torch"],
+        forceatlas=["forceatlas2-python"],
+        mkl=["mkl"],
+        all=["all"]
```

Comment:
What is `"all"` here?
The purpose of adding this `all` dependency list is to allow user use a one-line command to install all the dependencies (both required and optional) as before, i.e.
```
pip install pegasuspy[all]
```
will install everything.